### ISSUE TITLE
[ANDROID] PhonegapBuild - Error: IInAppBillingService does not exist

### ISSUE THREAD

**Issue by Ralpharoo** [j3k0/cordova-plugin-purchase#669]
I am having errors building with PGB similar too #659 even after linking to the current development build with the fix from this issue.

Config.xml;
```
<preference      name="phonegap-version"   value="cli-8.0.0" />
<plugin spec="https://github.com/j3k0/cordova-plugin-purchase" source="git">
    <param name="BILLING_KEY" value="..."/>
</plugin>
```

PGB log during Build;
>Fetching plugin "https://github.com/j3k0/cordova-plugin-purchase" via git clone
Using shallow clone
Repository "https://github.com/j3k0/cordova-plugin-purchase" checked out to git ref "master" at "12e9fc4".
Installing "cc.fovea.cordova.purchase" at "7.1.0" for android

Resulting error;
> BUILD FAILED in 3s
38 actionable tasks: 38 executed
Error: /gradlew: Command failed with exit code 1 Error output:
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
/app/src/main/java/com/smartmobilesoftware/util/IabHelper.java:33: error: package com.android.vending.billing does not exist
import com.android.vending.billing.IInAppBillingService;
                                  ^
/app/src/main/java/com/smartmobilesoftware/util/IabHelper.java:102: error: cannot find symbol
    IInAppBillingService mService;
    ^
  symbol:   class IInAppBillingService
  location: class IabHelper
/app/src/main/java/com/smartmobilesoftware/util/IabHelper.java:234: error: package IInAppBillingService does not exist
                mService = IInAppBillingService.Stub.asInterface(service);

Any suggestions how to fix this? Thanks in advanced.

**Comment by aesculus** [comment]
Any update on this? I need to send out a new version and cannot do that until this is fixed.  I also posted here but so far crickets.

https://forums.adobe.com/thread/2450497

**Comment by aesculus** [comment]
It appears that the target directory has changed in Android 7.0.0.

https://developer.android.com/google/play/billing/billing_integrate.html

This was recently addressed in the other IAP plugin

Can it be applied here please?

**Comment by Ralpharoo** [comment]
@aesculus - As per #659 they have updated the path I thought.

For now, I am reverting Android back;
```
<preference  platform="ios"      name="phonegap-version" 					value="cli-8.0.0" />
<preference  platform="android"  name="phonegap-version" 					value="cli-7.0.1" />
```

Then this also requires the camera to be rolled back;
```
<plugin platform="ios"      name="cordova-plugin-camera" source="npm" />
<plugin platform="android"  name="cordova-plugin-camera" spec="2.4.1" source="npm" />
```

Still, I really need 8.0.0 working as I am seeing crashes in this older camera plugin on Android Oreo which are already fixed.

**Comment by aesculus** [comment]
Bump. We still need this plugin updated to support PGB and to be placed in NPM too.

**Comment by aesculus** [comment]
This was fixed in V7.1.2 and verified with PGB CLI 8.0.0

**Comment by Ralpharoo** [comment]
Thanks @aesculus - confirmed fixed
